src/neo-vm/Types/Struct.cs

```diff
@@ -53,8 +57,11 @@ public Struct Clone()
                     }
                     else
                     {
+                        if (item is CompoundType ar)
+                            count -= ar.SubItemsCount;
```

Comment:
I think we should remove these 2 lines. Because if a `Struct` contains two identical arrays or maps, the number of their child elements will be counted repeatedly.

Reply:
@shargon Let's move on.